src/models/bulletChart.js

```diff
@@ -189,6 +194,7 @@ nv.models.bulletChart = function() {
         tickFormat:    {get: function(){return tickFormat;}, set: function(_){tickFormat=_;}},
         ticks:    {get: function(){return ticks;}, set: function(_){ticks=_;}},
         noData:    {get: function(){return noData;}, set: function(_){noData=_;}},
+        axisDuration:    {get: function(){return axisDuration;}, set: function(_){axisDuration=_;}},
```

Comment:
I just removed `axisDuration` and found a way to reuse the underlying bullet model `duration`. This way one can control both the bullet and axis duration with the standard `duration` parameter